ask/models.py

```diff
@@ -31,6 +31,8 @@ class question(models.Model):
     points = models.IntegerField(default=1)
     solved = models.BooleanField(default=False)
     tags = models.ManyToManyField(tag, blank=True)
+    flair_icon = models.CharField(max_length=25, null=True)
```

Comment:
I've added `flair-icon` so that we could use different flair icons for different type of flairs (gedditt?). This way we could specify _alert_, _verified_, and basically any the other icons from Material icons collection.